__tests__/app.test.js

```diff
@@ -167,3 +171,89 @@ describe("task 6. GET /api/users", () => {
       });
   });
 });
+
+describe("task 7. get /api/arcticles/:arcticle_id", () => {
+  test("status:200, responds with updated articles table with extra property", () => {
+    const article_id = 1;
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toEqual({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: "11",
```

Comment:
comment_count should be a number and not a string 
think about how we can change this (hint: look into this -->  :: INT)

Reply:
Will change this now! Thanks for the fb, sorry for putting on the same branch, i completely forgot i had done it all on the same one and panicked..